boards/px4/fmu-v3/init/rc.board_sensors

```diff
@@ -65,6 +65,9 @@ then
 	# external LSM303D is rotated 270 degrees yaw
 	lsm303d -X -R 6 start
 
+	# external emulated AK09916 (Here2) is rotated 270 degrees yaw
+	ak09916 -X -R 6 start
```

Comment:
Minor request, but can you group Here 1 and Here 2 together. It might also make sense to only attempt starting one if the other failed.

You should see it added here if you rebase.